app/static/main.js

```diff
@@ -2,7 +2,8 @@ let proxIdsSelected = new Set();
 let divIdsSelected = new Set();
 let selectedDatabaseId = null;
 let dbs = [];
-let prevInputValue = "";
+let prevTimeInputValue = "";
+let prevClusterInputValue = "";
```

Comment:
Change to `prevInputTimeValue` and `prevInputClusterValue` for consistent naming.